### PR TITLE
fix: fixed Blurred Booking Options Accessible #252

### LINKED ISSUE #252: HackTogether: Blurred Booking Options Accessible
**Are you accessing the project from the GitHub Codespaces?**

- [x] Yes, I am using GitHub Codespaces.

**Describe the bug**
User can edit booking form without the need to Login.

**To Reproduce**
1. Open the Portal
2. Click on any listing
3. Choose Options from the booking form

**Expected behavior**
The form isn't clickable if the user didn't login.

**Screenshots**
Listing Page Picking a Date
image

Listing Page Specifying number of guests
image

### PR DESCRIPTION
**Pull request by error9098x**
Fixes #252 Blurred Booking 

Solution:

- I added `[disabled]="isGuest()" ` to the affected `<input>` fields
- I added `pointer-events: none;` to `.filter-blur` scss

https://github.com/Azure-Samples/contoso-real-estate/assets/43810146/b882b89f-9c1d-4ea8-97cd-4acaabb32d5b

Again, I'm open to suggestions regarding other issues also I am open to other suggestions. :)

### CODE REVIEW

File: packages/portal/src/app/shared/booking-form/booking-form.component.html [hunk 1]
```diff
@@ -14,21 +14,21 @@
...
+      <input required matStartDate formControlName="start" placeholder="Start date" [disabled]="isGuest()" />
+      <input required matEndDate formControlName="end" placeholder="End date" [disabled]="isGuest()" />
     </mat-date-range-input>
     <mat-hint>MM/DD/YYYY – MM/DD/YYYY</mat-hint>
     <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
     <mat-date-range-picker #picker></mat-date-range-picker>
   </mat-form-field>
 
-  <mat-form-field appearance="outline">
+  <mat-form-field appearance="outline" >
```
Comment (anfibiacreativa) [inline packages/portal/src/app/shared/booking-form/booking-form.component.html:29]: Great accessibility fix! Thank you very much! Please remove the additional spaces between the closing brackets and the attributes and we're good to merge!

**Review by anfibiacreativa — CHANGES_REQUESTED** [review]
I love the accessibility perspective. Maybe you would like to try to add a Playwright test to this fix? That would be amazing! You could work together with @nitya to materialize it.

File: packages/portal/src/app/shared/booking-form/booking-form.component.html (hunk 1 above)
Comment (error9098x) [reply packages/portal/src/app/shared/booking-form/booking-form.component.html:29]: Sure !

**Comment by error9098x** [comment]
> I love the accessibility perspective. Maybe you would like to try to add a Playwright test to this fix? That would be amazing! You could work together with @nitya to materialize it.

@anfibiacreativa Sorry for the late commit for requested changes, I had to go to uni. I have removed the extra spaces from the lines in the PR. Regarding the Playwright test, I would really like to work on adding that - it's a great idea! Could you please let me know how I can connect with @nitya to collaborate on materializing it? I'm excited to learn more about implementing accessibility testing. Thank you so much for the thoughtful code review and suggestions! Let me know if you have any other feedback on the PR.

**Comment by anfibiacreativa** [comment]
> > I love the accessibility perspective. Maybe you would like to try to add a Playwright test to this fix? That would be amazing! You could work together with @nitya to materialize it.
> 
> @anfibiacreativa Sorry for the late commit for requested changes, I had to go to uni. I have removed the extra spaces from the lines in the PR. Regarding the Playwright test, I would really like to work on adding that - it's a great idea! Could you please let me know how I can connect with @nitya to collaborate on materializing it? I'm excited to learn more about implementing accessibility testing. Thank you so much for the thoughtful code review and suggestions! Let me know if you have any other feedback on the PR.

You don't have to apologize! Thank you so much for your contribution.

Regarding the test, you could open an issue describing what needs tested and assign it to @nitya so she can describe what she would like to see in the spec, test and then review it with you.

**Review by anfibiacreativa — APPROVED** [review]
Thanks for your contribution to enhance accessibility and usability in this project!

**Review by sinedied — APPROVED** [review]

**Pull request merged**